marlowe-chain-sync/deploy/asset.sql

```diff
@@ -0,0 +1,42 @@
+-- Deploy chain:asset to pg
+-- requires: tx
+
+BEGIN;
+
+CREATE TABLE chain.asset
+  ( id SERIAL
+  , policyId BYTEA NOT NULL
+  , name BYTEA NOT NULL
+  , CONSTRAINT pk_asset PRIMARY KEY (id)
+  , CONSTRAINT unique_asset UNIQUE (policyId, name)
+  );
+
+CREATE TABLE chain.assetOut
+  ( txOutId BYTEA NOT NULL
+  , txOutIx BIGINT NOT NULL
```

Comment:
Technically, we don't need `BIGINT`, even though this maps to `Integer`, because it is impossible for `txOutIx` to ever be greater than about 10k.

Reply:
Good to know!